kafka-client/src/main/java/dev/responsive/kafka/store/SchemaType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 Responsive Computing, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.responsive.kafka.store;
+
+public enum SchemaType {
+  KEY_VALUE,
+  IDEMPOTENT
```

Comment:
What about something like, `KEY_VALUE` vs `VALUE_AGNOSTIC` ...or even `VALUE_LESS`?

This is a little awkward but maybe you'll have better ideas along this line of thought

Reply:
I think @rodesai suggested `KeySet`, I think that might be a good thing to do here and just literally implement it as a key set. It still isn't 100% accurate because deletes can cause a zombie to corrupt data... 

`ResponsiveStores.keySetStore()` seems fine and `SchemaType.KEY_SET` is nice.

Reply:
I went with `WRITE_ONCE`

Reply:
OK I went with `FACT`. This use case is almost exactly summed up by the warehouse concept of a Fact store - it's not 100% in line with the definition but it's the best thing I could come up with, essentially it's expected to store "facts" where each key is expected to map to only a single value. Keys in fact tables are usually composite primary keys that reference something else (e.g. sensor, date -> measurement)